src/main.js

```diff
@@ -138,3 +143,29 @@ const killAllRunningProcesses = () => {
     console.error('Got error when trying to kill children', err);
   }
 };
+
+const showMoveToApplicationsFolderDialog = () => {
+  dialog.showMessageBox(
+    {
+      type: 'question',
+      buttons: ['Yes, move', 'Cancel'],
+      message: 'Move to applications folder?',
+      detail:
+        "I see that I'm not in the Applications folder. I can move myself there if you'd like!",
+      icon: path.join(__dirname, 'assets/icons/png/256x256.png'),
+      defaultId: 0,
```

Comment:
Can you add `cancelId: 1` to the options?

This way, when you escape out of the dialog, it'll select "Cancel"

(I _believe_ this is only necessary for Windows. Even though this is a MacOS prompt, it's still good to be consistent in case someone copies this code to a cross-platform place)